src/applications/simple-forms/26-4555/config/submit-transformer.js

```diff
@@ -0,0 +1,37 @@
+import sharedTransformForSubmit from '../../shared/config/submit-transformer';
+
+export default function transformForSubmit(formConfig, form) {
+  const remarksFormData = form.data.remarks;
+  const remarksUiSchema =
+    formConfig.chapters.additionalInformationChapter.pages.remarks.uiSchema
+      .remarks;
+
+  let remarkString = '';
+
+  Object.keys(remarksFormData).forEach(remarkKey => {
+    // filters by 'truthy' values, so non-empty strings and 'true' checkboxes
+    if (remarksFormData[remarkKey]) {
+      // 'true' checkboxes add their ui:title
+      // non-empty strings add their string content
+      remarkString +=
+        remarksFormData[remarkKey] === true
+          ? `${remarksUiSchema[remarkKey]['ui:title']}, `
```

Comment:
2 thoughts - non blocking
- could consider using reduce instead of forEach for this whole block, but this has each step which is explicitly nice. 
- if there was a way to check based on the data type (boolean) rather than title, that might be more future proof. but this solution seems fine too.

Reply:
Yeah I wasn't a fan of using forEach here either. It seemed the most clear to me to do it this way since there's some funky logic going on inside, like grabbing the titles for the booleans, but there's definitely a fancier way to do it with reduce or some other function. Maybe we can come back to this if we ever have to implement some funky logic like this in the submit controller again and clean it up.